episodes/2-identify-the-problem.Rmd

```diff
@@ -226,34 +201,36 @@ facet_wrap(~species)
 
 This looks much better, and is easier to see the trends over time as well. Note our x axis still shows bins with year labelings, but the continuous spread of our data over these bins shows that dates are treated more continuously and fall more continuously within histogram bins.
 
-:::::::: callout 
-One aspect we can see with this exercise above is that by setting up a reproducible example, we can isolate the problem with data rather than simply asking a proximal problem (i.e. 'how can i change my plot to look like so'). This allows helpers and you to directly improve your code, but also allows the community to help in identifying the problem. You don't always need to  understand what exact lines of code or function calls are going wrong in order to get help!
-::::::::
 
+One aspect we can see with this exercise above is that sometimes the 'problem' we need to solve is not super intuitive or can be isolated to one line. By setting up a reproducible example, we can see the isolated problem with real data rather than simply asking a proximal problem online or to a friend (i.e. 'how can i change my plot to look like so'). This allows the community to assist in identifying the problem -- you don't always need to have it figured out before asking for help! 
 
-## Summary
 
-In general, we need to isolate the specific areas of code causing the bug or problem. There is no general rule of thumb as to how large this needs to be, but in general, think about what we would want to include in a reprex. Any early lines which we know run correctly and as intended may not need to be included, and we should seek to isolate the problem area as much as we can to make it understandable to others. 
+:::::: callout
```

Comment:
I like this callout; it's a benefit I wouldn't have thought of highlighting, but it's absolutely true!